benchadapt/python/benchadapt/adapters/_adapter.py

```diff
@@ -162,10 +164,23 @@ def post_results(self) -> list:
 
         log.info("Posting results to conbench")
         res_list = []
+        error = None
         for result in self.results:
             result_dict = result.to_publishable_dict()
-            res = client.post(path="/benchmarks/", json=result_dict)
+
+            try:
+                res = client.post(path="/benchmarks/", json=result_dict)
+            except requests.exceptions.ReadTimeout as e:
+                print(f"POST timed out: {e.response.content.decode()}. Retrying...")
```

Comment:
There will be no response.

Reply:
You can print `e` and time it happens so it can be tracked in the conbench logs.

Reply:
Wait if i use `warnings.warn()` will those end up in the logs? That would be less gross than the `print()`s

Reply:
I really don't know and I never know unless I setup logging for the app myself :)